<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hangmanWidget</class>
 <widget class="QWidget" name="hangmanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblPlayerScore">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHiddenWord">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>129</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberOfGuesses">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberOfHintsLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hints Left:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtHint">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click this button to give you a random letter out of the word.&lt;/p&gt;&lt;p&gt;!!! THIS INCLUDES LETTERS THAT HAVE ALREADY BEEN GUESSED !!!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&amp;Hint</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerName">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerNameLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtQuit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>500</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneGuess">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a letter and click on &amp;quot;Guess&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Make A Guess</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerScoreLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Score:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtGuess">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter a letter in the text box to the right. Then click this button to make a guess.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&amp;Guess</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberOfGuessesLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Guesses Left:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberOfHints">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTimer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>02:00</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
